README.md

```diff
@@ -1,2 +1,46 @@
-# persistent-volume-migrator
-Tool to migrate Kubernetes In-tree and Flex volumes to Ceph-CSI
+# What is persistent-volume-migrator?
+
+The persistent-volume-migrator tool aims to help migrate Flex RBD Volumes to [Ceph-CSI](https://github.com/ceph/ceph-csi) RBD Volumes.
+
+# Getting Started
+
+**NOTE**: The procedure will come with downtime while the applications are scaled down for the volume migration.
+## Prerequisite
+This guide assumes you have created a Ceph cluster with Flex and [Ceph-CSI](https://github.com/ceph/ceph-csi).
+1. We assume that you have healthy cluster i.e. `ceph status` shows `health: OK`
+2. Minimum ceph version should be Ceph-CSI supported
+3. Stop the application pods that are consuming the flex volume(s) that need to be converted.
+4. Ceph-CSI storageClass should be created in which you want to migrate.
```

Comment:
no pre-req to set KUBECONFIG from where you will run this binary?

Reply:
I thought If the user have CSI which mean they have KUBECONFIG set and the user will run from the migrator pod(similar to the toolbox) which will have this set?